code/modules/organs/organ_external.dm

```diff
@@ -502,7 +502,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 
 		//Internal wounds get worse over time. Low temperatures (cryo) stop them.
 		if(W.internal && !W.is_treated() && owner.bodytemperature >= 170 && !(owner.species && owner.species.flags & NO_BLOOD))
-			if(!owner.reagents.has_reagent(BICARIDINE) && !owner.reagents.has_reagent(INAPROVALINE) && !owner.reagents.has_reagent(CLOTTING_AGENT) && !owner.reagents.has_reagent(BIOFOAM))	//Bicard, inaprovaline, clotting agent, and biofoam stop internal wounds from growing bigger with time, and also slow bleeding
+			if(!owner.reagents.has_reagent(BICARIDINE) && !owner.reagents.has_reagent(INAPROVALINE) && !owner.reagents.has_reagent(CLOTTING_AGENT) && !owner.reagents.has_reagent(BIOFOAM) && !owner.reagents.has_reagent(ECTOPLASM))	//Bicard, inaprovaline, clotting agent, biofoam, and ectoplasm stop internal wounds from growing bigger with time, and also slow bleeding
```

Comment:
I just made a `has_any_reagents()` proc so that all these could be passed at once as a list instead of calling `has_reagent()` individually on each one.